src/pages/NewsList/NewsList.tsx

```diff
@@ -44,52 +34,59 @@ export default function NewsList() {
 
 	useEffect(() => {
 		const fetchNewsData = async () => {
-			console.log('Fetch data: trying to fetch');
+			console.log('Fetch data: fetch started');
 			try {
 				if (!apiKey) {
 					throw new Error('API key not available');
 				}
 
-				const response = await fetch(`https://www.alphavantage.co/query?function=NEWS_SENTIMENT&tickers=IBM&limit=10000&apikey=${apiKey}`);
+				const response = await fetch(`https://www.alphavantage.co/query?function=NEWS_SENTIMENT&tickers=AAPL&limit=1000000&apikey=${apiKey}`);
```

Comment:
It would be a good idea to store this URL in some variable at the top of a file. So when it changes, you see it right away.